src/rabbit_stream_coordinator.erl

```diff
@@ -115,7 +115,9 @@ delete_replica(StreamId, Node) ->
     process_command({delete_replica, #{stream_id => StreamId, node => Node}}).
 
 process_command(Cmd) ->
+    global:set_lock(?STREAM_COORDINATOR_STARTUP),
```

Comment:
I don't know what the stream coordinator is, but do we need to acquire the lock on all connected nodes (as opposed to the local node only)?

Reply:
We do, because we could end up starting the whole cluster here. It will only affect declare/delete operations which if stream queues are used properly won't have a high churn